README.md

```diff
@@ -47,21 +47,28 @@ WP Rig can be used in any development environment. It does not require any speci
+In order to enable HTTPS with BrowserSync, you must supply a valid certificate and key with the Subject Alternative Name of `localhost`. Common Name has been deprecated since 2000 ([details](https://www.chromestatus.com/features/4981025180483584)).
 
-Once you have a key and certificate, uncomment the `certPath` and `keyPath` values in `dev/config/themeConfig.js` and update them to the absolute paths of your key and cert. You will also need to update `https` to `true` in `dev/config/themeConfig.js`.
+WP Rig can generate a key and certificate valid for `localhost` for you with the command `npm run generateCert`. The key and certificates will be saved as `BrowserSync/wp-rig-browser-sync-key.key`, `BrowserSync/wp-rig-browser-sync-root-cert.crt` and `BrowserSync/wp-rig-browser-sync-cert.crt` in the WP Rig theme directory.
 
-In addition, to disable SSL certificate warnings in the browser you will need to trust the certificate.
+To use the generated key and certificate across multiple projects copy `BrowserSync/wp-rig-browser-sync-key.key`, `BrowserSync/wp-rig-browser-sync-root-cert.crt` and `BrowserSync/wp-rig-browser-sync-cert.crt` in the WP Rig theme directory to a global location, such as your home directory, uncomment and update the `keyPath` and `certPath` values in `dev/config/themeConfig.js` with the absolute paths for the key and certificate.
+
+Then in future WP Rig projects you can simply set `https` to `true` and uncomment and update `keyPath` and `certPath` in `dev/config/themeConfig.js` with the paths of your existing key and certificate files rather than generating new ones for each project. Note that the certificate must be signed with an accompanying CA certificate and both certificates need to be trusted.
+
+If you leave `keyPath` and `certPath` commented out then the default paths of `BrowserSync/wp-rig-browser-sync-key.key` and `BrowserSync/wp-rig-browser-sync-cert.crt` in the WP Rig theme directory will be used. Once you have a key and certificate update `https` to `true` in `dev/config/themeConfig.js`.
+
+In addition, to disable SSL certificate warnings in the browser you will need to trust the certificate and CA certificate.
 
     * For macOS
         1. Open Keychain Access
-        2. Drag the certificate file onto KeyChain access
-        3. Double click on the new `WP Rig` certificate in the list
+        2. Drag the certificate files, which if using `npm run generateCert` to generate the certificates are saved to `BrowserSync/wp-rig-browser-sync-cert.crt` and `BrowserSync/wp-rig-browser-sync-root-cert.crt`, onto KeyChain access
```

Comment:
I didn't need to trust `BrowserSync/wp-rig-browser-sync-root-cert.crt`.

I made a couple screenshots that I'll place into this section.

Reply:
Also, when I dragged and dropped into Keychain Access, it just labelled it "localhost".

Reply:
Yeah, I don't think Keychain Access uses filename or anything just the hostname, which is `localhost`. That might be confusing if folks already have other `localhost` certificated but I don't think we can do anything about it.